appengine/endpoints/.gitignore

```diff
@@ -0,0 +1 @@
+endpoints/
```

Comment:
Does an endpoints/ subdirectory get used or generated by this sample?  Just wondering why to ignore this subdirectory